examples/pipe/interlock_test.c

```diff
@@ -104,7 +148,7 @@ static void *null_writer(pthread_addr_t pvarg)
 
 int interlock_test(void)
 {
-  pthread_t writerid;
+  pthread_t readerid, writerid;
```

Comment:
```suggestion
```suggestion
  pthread_t writerid;
  pthread_t readerid;
``````

Reply:
done!